### ISSUE TITLE
Update platform page Research and design left navigation

### ISSUE THREAD

**Issue by claytonzook** [department-of-veterans-affairs/va.gov-team#39779]
## User Story
"As a VFS teammate, I want to easily find the right page on Platform, so that I have quick access to the information that's important to me."

"As a platform team member, I want to be able to easily navigate to pages relevant for my practice area when I'm supplying links to answer questions"

Bet:
"As a VFS teammate, I want to set the the different practice areas on the left side navigation for the Research and design section of platform so I can more easily find information needed at a given time."

"As a new teammate, it would be easier to navigate by practice area to discover what I need to know for each."

- Intended Audience: VFS
- Desired Outcome: Easier to find information and to navigate on Platform's Research and design section
- Due Date: 

## Description

Current state on the [Research and design](https://depo-platform-documentation.scrollhelp.site/research-design/) section of platform, the left side navigation mostly links research pages.
Since the only way to find much of the design, IA, accessibility and QA information is to scroll down that section's home page, it can take additional effort to find needed information. Also since those links are research heavy, it makes this look like a section only for research.

Suggesting change - have left nav include a link to a page for each practice area. This will allow each practice area to better organize their information, and will be easier for VFS members to find needed information.

Additional background for [Launch new Platform Orientation](https://app.zenhub.com/workspaces/platform-governance-team-5f85b91c14d8df0018fac414/issues/department-of-veterans-affairs/va.gov-team/13709) initiative <br />
Additional background for [Collaboration Cycle - Improve tracking and status visibility of VA.gov products](https://app.zenhub.com/workspaces/platform-governance-team-5f85b91c14d8df0018fac414/issues/department-of-veterans-affairs/va.gov-team/34697) initiative <br />
Additional background for [Collaboration Cycle - Transition VFS research activities to Platform](https://app.zenhub.com/workspaces/platform-governance-team-5f85b91c14d8df0018fac414/issues/department-of-veterans-affairs/va.gov-team/32698) initiative

## Artifacts Potentially / Confirmed Needing Updates
[Platform: Research and design guide](https://depo-platform-documentation.scrollhelp.site/research-design/)
[Confluence: Research and design - place to edit the above platform page](Research and design )

## Tasks
- [ ] Consider best way to reorganize left nav
- [ ] Could include card sorting task at least for governance team, possibly including VFS team members as well

## Acceptance Criteria
- [ ] Updated left nav organization for [Platform: Research and design guide](https://depo-platform-documentation.scrollhelp.site/research-design/)

- [ ] "Insert Page Name" Confluence document has been updated per guidance
- [ ] "Insert Page Name" Confluence document has been created per guidance
- [ ] "Insert GitHub template" GitHub template has been created per guidance
- [ ] "Insert GitHub template" GitHub template has been updated per guidance
- [ ] "Insert Platform Website Page" on Platform Website has been updated per guidance
- [ ] "Insert Platform Website Page" on Platform Website has been created per guidance
- [ ] Updates have been Peer Reviewed by Governance Team member
- [ ] User Story need has been fulfilled

---

## Related Governance Team Resources
_For additional information and guidance to complete tasks, please see links below._
[Documentation Update Guidance for GitHub templates](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2047868956/Guidance+for+updating+GitHub+Issue+templates) <br />
[Documentation Update Guidance for Platform Website pages](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2089091115/Guidance+for+updating+Platform+Website+pages) <br />
[Documentation Update Guidance for Confluence pages](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2089222173/Guidance+for+updating+Confluence+pages) <br />
[Guidelines for Governance Team Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1127219219/Guidelines+for+Team+spaces) <br />
[Guidelines for Platform Products and Services Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1321140361/Guidelines+for+Platform+Products+and+Services+space) <br />
[Description of Platform Website spaces and content](https://vfs.atlassian.net/wiki/spaces/AP/pages/1241579796/Organization) <br />

---
## How to configure this issue
- [ ] **Ensure that story is attached to appropriate Iniative (if not tech debt)** (what body of work is this a part of?)
- [ ] Ensure that any additional labels are added ('tech-debt', 'collaboration-cycle')
- [ ] Ensure that story is in the correct pipeline (Backlog, Planned, In Progress, Review/QA, Closed)
- [ ] Ensure that assignees are correct so you receive all updates/comments on the ticket

**Comment by shiragoodman** [comment]
@naomimarcussen let's talk about this when you're back in the office :)

@claytonzook thank you for writing this up!

**Comment by naomimarcussen** [comment]
This is great @claytonzook! We definitely need to update this section. @shiragoodman I'm back for now. I'd love to talk about this. If you schedule a meeting we can always reschedule if things change with my availability.

**Comment by claytonzook** [comment]
copy/pasted from [ticket 39472](https://app.zenhub.com/workspaces/platform-governance-team-5f85b91c14d8df0018fac414/issues/department-of-veterans-affairs/va.gov-team/39472)
___________
claytonzook
commented on Apr 12
After thinking about this more and after talking with the Content/IA team

I've changed this ticket to focus on the Research and design section and taken out the Collab Cycle section
Content/IA team verified that there was no present objection with our area pages including content along with the links, differing from the other 4 main area pages that just have links and no content.
Content/IA team agreed that the next steps for this would be research to further consider and validate best way forward on nav link changes.

**Comment by shiragoodman** [comment]
just my 2 cents, I think change option 2 makes more sense as I don't really see anything currently in the left nav that is practice-area specific. My advice would be to organize by phase of research with a dedicated section for the Overview. The phases can be Preparing for Research, During Research and Post Research (or other names as appropriate). I'd still like to hear @jimuhh's opinion though.

Keep in mind that we won't be _doing research_ to determine the best left nav organization, however we should be making an assumption of how we think VFS researchers use this page, and implementing an organization based on that. If/when this section of Platform Website continues to grow and the organization model we've implemented is no longer sufficient, then we may do research at that time to determine how better to organize the left nav.

**Comment by claytonzook** [comment]
@shiragoodman Thinking about this more, I agree with your assessment on option 2. Although I do feel weird about the left nav links being focused on research since the content of this page covers so much more than that. 

Thinking through option 1 at little more - similar to how we have been moving remaining research area GH docs to Platform creating this left nav problem, some GH docs for other practice areas could benefit from that move as well. So this is definitely more of a solution to a problem we don't currently have, but I'd still hope that we have down the road.

RE research, I would be remiss if I didn't say I think we should research. But I think we can discuss further elsewhere to better effect.

**Comment by naomimarcussen** [comment]
@claytonzook and @shiragoodman I think we eventually want to move toward option 1, but proceeding with option 2 in the meantime will be a good first step. I'd like to gather Governance folks for a workshop where we work to create a plan toward that goal.

**Comment by jimuhh** [comment]
Here's [a draft revised navigation based on the card sort results](https://docs.google.com/spreadsheets/d/1NwzEZLDbsRnM9lbBxn_jiftwn5XkvzCu69VNC8au7Uk/edit?usp=sharing). I'd like to tree test it to make sure it's good enough.

**Comment by jimuhh** [comment]
Some of the nav items:

- Best practice: Focus management
- Best practice: Accessible links
- Best practice: Buttons vs Links

Seem out of place in this nav schema. Any chance we can relocate them?

**Comment by naomimarcussen** [comment]
@jimuhh The intention is for this section to house information for doing both research and design. At the moment we only have documentation available for research.

**Comment by shiragoodman** [comment]
@naomimarcussen I didn't know that! but if that's the case, then I'm supportive of keeping the name (although I do agree with Jim).

@jimuhh re: your question about the out of place "Best practice" links. I agree. They do seem out of place, especially since this page is currently focused on Research. @claytonzook - do you think these 3 pages make sense under the Research and Design tile? I'm not sure I see the direct connection between accessibility best practices and conducting research. Maybe we should relocate to the [Accessibility Testing page](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-testing), for now.

**Comment by jimuhh** [comment]
@claytonzook Given that we can't change this page without reworking the landing page, which might be a lengthy discussion, I think we can probably get away with separating out the practice areas here tho' it's far from ideal. You're right that there aran't a lot of 4th level pages in the current research schema. Hopefully that will also apply to the proposed design navs as well.

**Comment by jimuhh** [comment]
@claytonzook @naomimarcussen @shiragoodman but it we do bifurcate the leftnav by practice area as described, we should do it from the git-go, even if design content isn't yet in place. Otherwise, we're going to be moving a lot of content down in the hierarchy later on, which could get confusing.

**Comment by jimuhh** [comment]
See where the "architecture" part comes in? You plan to remodel one room and suddenly you're moving retaining walls and pulling out  plumbing.

**Comment by jimuhh** [comment]
@shiragoodman I had considered this approach and it may turn out to be the best available solution, but it does break the expected nav pattern:

- **Category** (e.g. Design)
  - Item (e.g. using the design system)
  

Putting categories at the Item level increases cognitive load by making users read each item to figure out its category instead of just looking at the bucket it's in. 

This might also result in a _very_ long leftnav if and when the promised design content shows up. Also, by adding the "Research" and "Design" modifiers to nav items we'd be breaking our own rule about matching nav labels with the H1 of target pages and making each nav item a longer read. Adding the category and colon to our existing breadcrumb pattern, which is based on the category/item hierarchy, would also represent a significant change. 
Thoughts?

**Comment by jimuhh** [comment]
On second thought, there are only 4 1st level categories in this layout, so while I don't love ganging up categories in item-level labels, we could probably add the same number for the new design content without overcrowding the nav. And also found there's [precedent for the colon pattern in breadcrumbs on FAQ pages](https://staging.va.gov/coronavirus-veteran-frequently-asked-questions/#more-coronavirus-specific-conc) so we can get away w/ this.

**Comment by shiragoodman** [comment]
@jimuhh right now, there's 12 1st level categories, so I think a max of 8 wouldn't be too cumbersome. I'd like to loop in Shane to get his opinion since Clayton is out of office. Could you put together a finalized design to share with Shane (and Naomi too) for their review and hopefully approval?

**Comment by jimuhh** [comment]
@shiragoodman sure. I'll just add that to the existing spreadsheet.

**Comment by jimuhh** [comment]
@shiragoodman, @shanemelliott, @naomimarcussen 
[Here is the revised nav layout](https://docs.google.com/spreadsheets/d/1NwzEZLDbsRnM9lbBxn_jiftwn5XkvzCu69VNC8au7Uk/edit?usp=sharing) with "Research:" added to top level items. 
The idea is that when design content becomes available, we'll treat it the same way, with several categorized top level navs that expands on click to show subnavs like this. 
- **Design: Using the design system** 
  - Intro to design system
  - Foundations, Components and Patterns
  - etc.
  - etc.

**Comment by jimuhh** [comment]
@shanemelliott, @naomimarcussen 
LMK if you have any questions/concerns.

**Comment by naomimarcussen** [comment]
Thank you for organizing this! I like this approach. It got me thinking: 
- Is there a reason you pulled out just the "Observer guidelines" and "Pilot participant guidelines"? 
- In the "Research: Planning and recruiting" section, what is the difference between Planning research and the checklist.
- We have headers for only part of the process and I fear people won't see what we want them to do to share their findings. What if we add "Synthesize and Share" as the last header so it reads as a full process? 
- Breaking it out by a process has me wondering if breaking out the "Research checklist" into smaller parts and have that at the top of each section is a way to connect the processes and resources we have there with other supporting content. I'm sure there are better ways to name these though. What do you think?
- As for the zoom, iPhone and Android pages, we could move the zoom header into the "Conducting sessions" section. Then, what if we organize the rest by mobile and assistive technology?
- I added [another tab](https://docs.google.com/spreadsheets/d/1NwzEZLDbsRnM9lbBxn_jiftwn5XkvzCu69VNC8au7Uk/edit#gid=1862257273) to the spreadsheet to sketch out my thoughts.

For another ticket:
In addition to the research and design sections, we also need a "Getting started" section that has all the info we provide in our orientation deck like: 
- Who we are
- How to contact us
- How we collaborate with links to design office hours and the weekly design meeting
- Design principles
- Overview of our UX process

**Comment by shiragoodman** [comment]
@jimuhh looks like you tagged the wrong Shane! no worries, it happens. FYI @sstrassberg

**Comment by jimuhh** [comment]
- In the "Research: Planning and recruiting" section, what is the difference between Planning research and the checklist.
  - **Good question**. If the concern is whether they should be rolled together, I can't say. The ask on this project was to rework the leftnav, using the content that's there already. 
  - Questions about what pages should be there and/or what content they should feature, though totally valid, are one are better decided with a content inventory and audit, which weren't in scope for this ticket.

**Comment by jimuhh** [comment]
Hi @naomimarcussen. Herewith...
- We have headers for only part of the process and I fear people won't see what we want them to do to share their findings. What if we add "Synthesize and Share" as the last header so it reads as a full process?
  - I would totally endorse that. 
  - AFAIK, the content that people card-sorted wasn't originally intende to represent an end-to-end process, but now that we know we'd like that, it makes total sense to add new categories and content to that end.

**Comment by jimuhh** [comment]
Hi @naomimarcussen. continued...
- Breaking it out by a process has me wondering if breaking out the "Research checklist" into smaller parts and have that at the top of each section is a way to connect the processes and resources we have there with other supporting content. I'm sure there are better ways to name these though. What do you think?
  - I could totally see the value in doing that. 
  - Even though content inventory/audit weren't part of original scope, it seems the card sort is leading us to those kinds of discussions and decisions as so often happens. 😁

**Comment by jimuhh** [comment]
- As for the zoom, iPhone and Android pages, we could move the zoom header into the "Conducting sessions" section. Then, what if we organize the rest by mobile and assistive technology?
  -  That makes sense to me, but the schema I shared just represents the fact that most people grouped those mobile-related items together as a separate category. 
  - I honestly don't know if people grouped the mobile tech help content together simply because there was so much of it. All I know is that they did. 🤷‍♂️

**Comment by jimuhh** [comment]
@naomimarcussen 
I like your version and I'm glad that these results are prompting some "what ifs." It would be worth our while to turn our current best estimate of what an improved leftnav looks like into a tree test that includes some of the content revisions you suggested. 
If you like that idea, it would be great to hear some suggestions about task scenarios we might use for testing.

**Comment by jimuhh** [comment]
@shiragoodman and if @naomimarcussen agrees that we should tree test the new revised nav  before implementing, how would you feel about capturing that in a new ticket and closing this one?

**Comment by jimuhh** [comment]
Thanks @shiragoodman! I would have been OK with a slightly re-grouped leftnav for now but based on @naomimarcussen's feedback,  it seems like this exercise is raising a lot of legitimate content strategy questions.

**Comment by jimuhh** [comment]
@shiragoodman. I'm ready to close this now. @naomimarcussen is out this week but that wouldn't stop me from turning her iteration of this nav into a tree test to discuss when she gets back.

**Comment by shiragoodman** [comment]
@jimuhh can you please update the tasks and AC of this ticket? I want to make sure I know what work was completed in this ticket and what should be moved to #44196. Once that's complete, we can close this out.

**Comment by jimuhh** [comment]
@shiragoodman Will do. Thx.

**Comment by jimuhh** [comment]
@shiragoodman -- The only outstanding task is

-  Ensure no links have been broken in this change
...because obviously that could only happen if the revised nav went to staging, so everything else is OK to close. 
I wonder whether we shouldn't break out the final production of leftnav to to the next ticket after #44196 since the tree test on next phase might reveal that our planned nav revision doesn't actually work for users.

**Comment by shiragoodman** [comment]
thanks, @jimuhh. I will hold off on closing this ticket until #44196 is ready.

**Comment by shiragoodman** [comment]
hey @jimuhh  - sorry, I was reviewing this ticket and it's still not where it needs to be. My 7/12 comment was intended for you to update the tasks and AC with the work that was actually done for this ticket. For example, we did not, "Updated left nav organization for Platform: Research and design guide" so that AC should be removed. What I'd like to have in the AC is what was completed. Possibly, "We have completed a card sort for the existing pages of the left nav." Aside from that, what else did we do?

Let me know if you'd like to meet to discuss.

**Comment by jimuhh** [comment]
@shiragoodman Ah OK. Thanks for clarifying. I can do that.

**Comment by jimuhh** [comment]
@shiragoodman OK. I brought the AC in line w/ the revised workflow (how very agile of me! 😀 ) 
LMK if this works for you.

**Comment by claytonzook** [comment]
@jimuhh 
I'm a little late to the party, but my largest concern on the current redo (as listed in https://github.com/department-of-veterans-affairs/va.gov-team/issues/39779#issuecomment-1178163602) is that Research Checklist is a whole end to end research page (ie Overview), but not listed with that grouping. It's an intriguing idea to split this out into multiple checklists, because in reality this checklist is best completed in chunks. To say, I'm along for this ride.

> * Breaking it out by a process has me wondering if breaking out the "Research checklist" into smaller parts and have that at the top of each section is a way to connect the processes and resources we have there with other supporting content. I'm sure there are better ways to name these though. What do you think?

https://github.com/department-of-veterans-affairs/va.gov-team/issues/39779#issuecomment-1180907167
